README.md

```diff
@@ -97,6 +97,10 @@ representing the user choice (resolved on confirmation and rejected on cancellat
 | **`allowClose`**                        | `boolean`   | `true`                  | Whether natural close (escape or backdrop click) should close the dialog. When set to `false` force the user to either cancel or confirm explicitly.                                                                                   |
 | **`confirmationKeyword`**               | `string`    | `undefined`             | If provided the confirmation button will be disabled by default and an additional textfield will be rendered. The confirmation button will only be enabled when the contents of the textfield match the value of `confirmationKeyword` |
 | **`confirmationKeywordTextFieldProps`** | `object`    | `{}`                    | Material-UI [TextField](https://mui.com/material-ui/api/text-field/) props for the confirmation keyword textfield.                                                                                                                     |
+| **`isAcknowledgeCheckbox`**               | `boolean`    | `false`             | If enabled, it shows the acknowledge checkbox and disables the confirm button while the checkbox is unchecked.|
+| **`acknowledgeCheckboxLabel`**               | `string`    | `Please confirm`             | Acknowledge checkbox label |
```

Comment:
I would merge these two options. Let's make it `acknowledgement` string that is the actual label and enables the feature (or perhaps it should be arbitrary node, since Material UI allows that).

Then we have `acknowledgementFormControlLabelProps` and `acknowledgementCheckboxProps`.